### PR TITLE
Mitigating foxnews for users to create an account

### PR DESCRIPTION
**Pull request by icodebyamanda**
@jonathanKingston can you review and merge?

### CODE REVIEW

**Review by SlayterDev — CHANGES_REQUESTED** [review]
I don't see the issue immediately but the build JSON appears to be invalid. Please double check your change.

File: features/tracker-allowlist.json [hunk 1]
```diff
@@ -296,6 +296,17 @@
                     }
                 ]
             },
+            "segment.io": {
+                "rules": [
+                    {
+                        "rule": "cdn.segment.com/analytics.js/v1/4I2fAtE5IxTlczNT3F91OckHtI5wfR7g/analytics.min.js",
+                        "domains": [
+                            "foxnews.com",
```
Comment (SlayterDev) [inline features/tracker-allowlist.json:304]: You have a hanging comma here which is causing the validation error.

**Review by SlayterDev — CHANGES_REQUESTED** [review]

**Review by icodebyamanda — COMMENTED** [review]
L304 comma removed

File: features/tracker-allowlist.json (hunk 1 above)
Comment (icodebyamanda) [reply features/tracker-allowlist.json]: @SlayterDev sorry - I removed the comma.

**Review by SlayterDev — APPROVED** [review]

**Pull request merged**